core/common/src/ThematicDisplay.ts

```diff
@@ -341,13 +357,13 @@ export interface ThematicDisplayProps {
   /** For [[ThematicDisplayMode.HillShade]], this is the direction of the sun in world space. Defaults to {0,0,0}. */
   sunDirection?: XYZProps;
   /** For [[ThematicDisplayMode.InverseDistanceWeightedSensors]], these are the settings that control the sensors. Defaults to an instantiation using [[ThematicDisplaySensorSettings.fromJSON]] with no arguments.
-   * @alpha
+   * @public
    */
   sensorSettings?: ThematicDisplaySensorSettingsProps;
 }
 
 /** The thematic display setup of a [[DisplayStyle3d]].
- * @beta
+ * @public

```

Comment:
Can you add some description of what "thematic display" means / what it's useful for?

Reply:
Good suggestion.  Done.